src/components/form-elements/checkbox/Checkbox.jsx

```diff
@@ -213,17 +220,25 @@ const Checkbox = ({
 
   let checkboxIcon = null;
 
-  if (isChecked) checkboxIcon = <CheckIcon />;
+  if (isChecked && !indeterminate) checkboxIcon = <CheckIcon ref={iconRef} />;
+  if (indeterminate) checkboxIcon = <IndeterminateIcon ref={iconRef} />;
 
-  if (indeterminate) checkboxIcon = <IndeterminateIcon />;
+  React.useEffect(() => {
+    requestAnimationFrame(() => {
+      if (iconRef.current && iconRef.current instanceof window.SVGElement) {
```

Comment:
`iconRef.current instanceof window.SVGElement` is to satisfy flow?

Reply:
Yes. In fact we should use `SVGElement` type in the CheckboxIcon, but because flow doesn't recognise such type, we had to use `Element` there